README.md

```diff
@@ -33,7 +34,11 @@ Customisable smart app banner for iOS and Android.
 <meta name="smartbanner:icon-google" content="https://url/to/google-play-icon.png">
 <meta name="smartbanner:button" content="VIEW">
 <meta name="smartbanner:button-url-apple" content="https://ios/application-url">
+<meta name="smartbanner:button-url-apple-iphone" content="https://ios-iphone/application-url">
```

Comment:
Let's move all these specific cases out of _Basic Usage_ to _Advanced Usage_. We'd like to keep the basic block as simple as possible for easy entry.